src/api/helpers/text.cpp

```diff
@@ -35,6 +35,7 @@
 #endif
 
 using std::regex;
+using icu::UnicodeString;
```

Comment:
This breaks the Windows build, as it doesn't use ICU. If a `using` statement is needed, it should go before the `#endif` on line 35.

Reply:
Ah so that's why it failed, moved it